modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/indexreader/IgniteIndexReader.java

```diff
@@ -574,6 +581,9 @@ private Map<Integer, List<Throwable>> checkParts(Map<String, TreeTraversalInfo>
 
         ProgressPrinter progressPrinter = new ProgressPrinter(System.out, "Checking partitions", partCnt);
 
+        // Map<partNum, partItemsCnt>
```

Comment:
I'd rather see something like `{partNum -> partItemCnt}` in the comment. Type parameters should contain types, and their alternative usage looks inappropriate

Reply:
fixed